src/index.js

```diff
@@ -9,10 +9,14 @@ export default ({ form, fieldsToProps = {}, inspectorKey }) => {
     const FormInspectorComponent = BaseComponent => (
         connect((state) => {
             const fieldValues = getFormValues(form)(state);
+            const errors = {
```

Comment:
Not optimal as sync errors would override async ones one the same field, but let keep it simple for now.